server/proxy/proxy.go

```diff
@@ -42,6 +43,17 @@ func (p *Proxy) Call(sessionID, actingUserID string, creq *apps.CallRequest) *ap
 	if err != nil {
 		return apps.NewProxyCallResponse(apps.NewErrorCallResponse(err), metadata)
 	}
+
+	if creq.Path == "" || creq.Path[0] != '/' {
```

Comment:
Also, what's wrong with a "/" as a call path? I know it is unusual, but why not?